packages/neon-core/src/sc/manifest/WildCardContainer.ts

```diff
@@ -0,0 +1,35 @@
+export class WildCardContainer {
```

Comment:
This class is not necessary a wildcard if it contains data? I dont think having a container class is the right way to do this. Just stick to strings instead.

Reply:
Removed